steps/fleet/fleet.go

```diff
@@ -71,6 +71,23 @@ func Setup(fsc *fs.FsClient, sc *systemd.SystemdClient, fc fetchclient.FetchClie
 	return nil
 }
 
+func createFleetService(fsc *fs.FsClient, useOverlay bool) error {
```

Comment:
Aligned the way how the service template is created with how it is done in the docker step.